Content.Client/_DV/Curation/UI/Cwoink/CwoinkControl.xaml.cs

```diff
@@ -0,0 +1,282 @@
+            if (IsNewPlayer(info))
+                sb.Append(new Rune(0x23F2)); // ⏲
+
+            sb.AppendFormat((string)"\"{0}\"", (object?)text);
+
+            return sb.ToString();
+        };
+
+        // <summary>
+        // Returns true if the player's overall playtime is under the set threshold
+        // </summary>
+        bool IsNewPlayer(PlayerInfo info)
+        {
+            // Don't show every disconnected player as new, don't show 0-minute players as new if threshold is
+            if (newPlayerThreshold <= 0 || info.OverallPlaytime is null && !info.Connected)
+                return false;
+
+            return (info.OverallPlaytime is null
+                    || info.OverallPlaytime < TimeSpan.FromMinutes(newPlayerThreshold));
+        }
+
+        ChannelSelector.Comparison = (a, b) =>
+        {
+            var ach = CHelpHelper.EnsurePanel(a.SessionId);
+            var bch = CHelpHelper.EnsurePanel(b.SessionId);
+
+            // Pinned players first
+            if (a.IsPinned != b.IsPinned)
+                return a.IsPinned ? -1 : 1;
+
+            // Then, any chat with unread messages.
+            var aUnread = ach.Unread > 0;
+            var bUnread = bch.Unread > 0;
+            if (aUnread != bUnread)
+                return aUnread ? -1 : 1;
+
+            // Then, any chat with recent messages from the current round
+            var aRecent = a.ActiveThisRound && ach.LastMessage != DateTime.MinValue;
+            var bRecent = b.ActiveThisRound && bch.LastMessage != DateTime.MinValue;
+            if (aRecent != bRecent)
+                return aRecent ? -1 : 1;
+
+            // Sort by connection status. Disconnected players will be last.
+            if (a.Connected != b.Connected)
+                return a.Connected ? -1 : 1;
+
+            // Sort connected players by whether they have joined the round, then by New Player status, then by Antag status
+            if (a.Connected && b.Connected)
+            {
+                var aNewPlayer = IsNewPlayer(a);
+                var bNewPlayer = IsNewPlayer(b);
+
+                //  Players who have joined the round will be listed before players in the lobby
+                if (a.ActiveThisRound != b.ActiveThisRound)
+                    return a.ActiveThisRound ? -1 : 1;
+
+                //  Within both the joined group and lobby group, new players will be grouped and listed first
+                if (aNewPlayer != bNewPlayer)
+                    return aNewPlayer ? -1 : 1;
+
+                //  Within all four previous groups, antagonists will be listed first.
+                if (a.Antag != b.Antag)
+                    return a.Antag ? -1 : 1;
+            }
+
+            // Sort disconnected players by participation in the round
+            if (!a.Connected && !b.Connected)
+            {
+                if (a.ActiveThisRound != b.ActiveThisRound)
+                    return a.ActiveThisRound ? -1 : 1;
+            }
```

Comment:
This check is the same regardless of connection state, you can move the `a.ActiveThisRound != b.ActiveThisRound` before `a.Connected && b.Connected` and remove it from inside that block, then remove the entire `!a.Connected && !b.Connected` block